packages/docs/docs/tutorial/part4-docsqa.md

```diff
@@ -26,6 +26,7 @@ that are individually processed by the LLM. Each chunk is passed into the LLM,
 and the _text embedding_ representing the chunk is computed. (For more details on text
 embeddings, check out the [OpenAI text embeddings API docs](https://platform.openai.com/docs/guides/embeddings).) The text embedding is a multidimensional vector, calculated by the LLM,
 representing the semantic content of the chunk. A _corpus_ consists of a set of document chunks and their corresponding embedding vectors.
+For a more detailed explanation of how a corpus is prepared and used, see the guide [DocsQA: Grounding Answers with a Source of Truth](https://docs.ai-jsx.com/guides/docsqa#overview).
```

Comment:
Use a relative link to the markdown file in the monorepo, rather than an absolute link to the hosted file. That'll be more robust.

Reply:
Oops, nice catch.